students/alinafe/session03/rot13.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+"""
+The task
+Add a python module named rot13.py to the session03 dir in your student dir.
+This module should provide at least one function called rot13 that takes any
+amount of text and returns that same text encrypted by ROT13.
+
+This function should preserve whitespace, punctuation and capitalization.
+
+Your module should include an if __name__ == '__main__': block with tests (asserts)
+ that demonstrate that your rot13 function and any helper functions you add work properly.
+
+ordinals…
+“Ordinals” are the numerical values associated with characters. Python stringsl are native unicode,
+so the number values of any character
+
+Hints
+Note that the alphabet has 26 letters, so if you “rotate” by 13 letters twice, you will be back were you started.
+So if you call your function twice on a string, you should get the same string back.
+
+rot13(rot13(something)) == something
+
+There is a “short-cut” available that will help you accomplish this task. Some spelunking
+in the documentation for strings should help you to find it. If you do find it, using it is completely fair game.
+
+As usual, add your new file to your local clone right away. Make commits early and often
+and include commit messages that are descriptive and concise.
+
+When you are done, if you want me to review it, push your changes to github and issue a pull request.
+
+Try decrypting this:
+
+“Zntargvp sebz bhgfvqr arne pbeare”
+
+"""
+
+#ord('A') = 65
+#ord('Z') = 90
+#ord('a') = 97
+#ord('z') = 122
+
+
+def setn(x, low, high):
```

Comment:
interesting -- I don't think I've seen it done quite this way before, but it sure works.